mondrian/src/it/java/mondrian/rolap/NativeEvalVirtualCubeTest.java

```diff
@@ -362,7 +362,7 @@ public void testShouldUseCache() {
       + "group by\n"
       + "    `customer`.`gender`\n"
       + "order by\n"
-      + "    ISNULL(`customer`.`gender`) ASC, `customer`.`gender` ASC";
+      + "    ISNULL(`c0`) ASC, `c0` ASC";
```

Comment:
Not sure this is a legit change. Why did this change?

Reply:
This is because MySQL went from version 5.6 to 8.0;
https://github.com/pentaho/mondrian/blob/master/mondrian/src/main/java/mondrian/spi/impl/MySqlDialect.java#L359

The newer version of MySQL requires aliases in the order by statement.